render_dashboard.js

```diff
@@ -11,10 +11,8 @@
 // Some constants.
 //
 
-var DOCUMENT_WIDTH = 720;
-var DOCUMENT_HEIGHT = 3000;
-
-
+var DOCUMENT_WIDTH = (screen.width*200)/100;
+var DOCUMENT_HEIGHT = (screen.height*200)/100;
```

Comment:
I am not sure I understand the logic behind this change.

should we just make width/height configurable?